notebooks/MAST/m_dwarf_tess/m_dwarf_flares_tess.ipynb

```diff
@@ -0,0 +1,1285 @@
+{
```

Comment:
<p>This is a great intro. Make sure to add some links to background information about M Dwarfs. Also it would be cool to link to a paper on the habitability of planets around M Dwarfs, my quick google search yielded this paper: <a href="https://doi.org/10.48550/arXiv.2203.10127" target="_blank">https://doi.org/10.48550/arXiv.2203.10127</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>